### ISSUE TITLE
could not find function "renameTaxa"

### ISSUE THREAD

**Issue by JBL05** [stefpeschel/NetCoMi#83]
Hello, I am following the tutorial, but encountered the issue of "could not find function "renameTaxa", how can I get this function to work?

**Comment by stefpeschel** [comment]
Hey. 
This function has been implemented with the latest release 1.1.0. 
So, please update NetCoMi to the latest version using
```
devtools::install_github("stefpeschel/NetCoMi", 
                         dependencies = c("Depends", "Imports", "LinkingTo"),
                         repos = c("https://cloud.r-project.org/",
                                   BiocManager::repositories()),
                         force = TRUE)
```
You should then be able to use the `renameTaxa()` function.

Best, 
Stefanie

**Comment by JBL05** [comment]
Thank you very much!
